styleguide/derek/examples/_office365.html

```diff
@@ -2151,4 +2151,66 @@ <h4 class="center">The best part about all of our plans?<br/>
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+
+  <!-- Enterprise Managed Services -->
+   <div class="container-fluid feature-container bg-crumpled-paper">
+
+      <div class="row">
+        <!-- Left Column -->
+        <div class="col-md-5 no-padding">
+            <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/office-365/sidebyside.png">
+        </div>
+
+        <!-- Right Column -->
+        <div class="col-md-7">
+          <div class="col-md-11 half-padding">
+              <h2>Managed Services for Office 365.</h2>
+              <p>Maximize your Office 365 investment with advisory services and Enterprise-class support for Office 365. Get more out of your Office 365 investment.</p>
+          </div>
+          <!-- Feature 1 -->
+          <div class="col-md-12 feature">
```

Comment:
shouldnt need to create any new styling so the "feature" class is probably not necessary.